src/StockAvailabilityChecker.php

```diff
@@ -51,15 +51,17 @@ public function check(PurchasableEntityInterface $entity, $quantity, Context $co
     }
```

Comment:
Microperformanceoptimization. If we we need to return, do it before we request stockConfig and fetch the locations